framework/src/test/java/uk/gov/ons/ctp/common/cloud/FirestoreDataStoreTest.java

```diff
@@ -29,13 +30,15 @@
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.api.parallel.ResourceLock;
 import org.mockito.Mock;
 import org.mockito.Mockito;
 import org.mockito.junit.jupiter.MockitoExtension;
 import org.springframework.test.util.ReflectionTestUtils;
 import uk.gov.ons.ctp.common.error.CTPException;
 
 @ExtendWith(MockitoExtension.class)
+@ResourceLock(value = "FirestoreDataStoreTest", mode = READ_WRITE)
```

Comment:
this can be removed if you remove the "static" keyword on firestoreDataStore as mentioned below

Reply:
Agree. Changing